package.json

```diff
@@ -61,5 +61,19 @@
     "lint-staged": "^10.5.2",
     "prettier": "^2.2.1",
     "react-test-renderer": "^17.0.1"
+  },
```

Comment:
Probably need to exclude index here also.

Reply:
Thanks for all the updates and cleaning up the return statements and versions and test packages